Flutter/pdf-viewer/sticky-notes.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: Sticky Note Annotations in Flutter PDF Viewer Widget | Syncfusion
+description: Learn here all about sticky note annotations in the Syncfusion Flutter PDF Viewer (SfPdfViewer) widget and more.
+platform: flutter
+control: SfPdfViewer
+documentation: ug
+---
+
+# Sticky Note Annotations in Flutter PDF Viewer Widget (Syncfusion)
+
+The sticky note annotation feature of `SfPdfViewer` allows you to add, remove, and modify sticky notes in a PDF document. This feature will help add comments or notes to specific parts of a document to clarify complex concepts, terms, or ideas. This section will cover the various functions available in `SfPdfViewer` for working with sticky note annotations.
+
+## Types of sticky notes
+
+The following sticky note icon types are currently available in `SfPdfViewer`. The icon types can be selected from the [PdfStickyNoteIcon](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/PdfStickyNoteIcon.html) enumeration.
+
+![Sticky note icon types](images/annotations/sticky-note-icon-types.png)
+
+## Add sticky notes
+
+This section will go through how to add sticky note annotations to a PDF document interactively and programmatically.
+
+### Add sticky notes from annotation mode
```

Comment:
In some places API is linked, and in some other places it was not linked

<img width="759" alt="image" src="https://github.com/user-attachments/assets/c30406e6-22f0-4944-ae0b-0af9b73bd00b">

Reply:
API url linked